src/models/currency.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
+
+const currencySchema = mongoose.Schema({
+
+    currencyName: {
+        type: String,
+        require: true,
+        unique: true
+    },
+
+    ratio: {
+        type: Number,
+        require: true,
+    },
+
+    availableAmount: {
+        type: Number,
+        require: true
```

Comment:
```suggestion
        required: true
```
There is also the same mistake in the users model, if you can also correct the parameter there